src/core/services/utility.js

```diff
@@ -26,6 +28,9 @@ export {
 	isUndefined,
 	clone,
 	debounce,
+	merge,
+	yes,
```

Comment:
true|false is system stuff don't wont to mix it, truthy vs yes can make vote in the future)